rskj-core/src/main/java/co/rsk/peg/federation/FederationSupport.java

```diff
@@ -256,6 +256,11 @@ int voteFederationChange(
         SignatureCache signatureCache,
         BridgeEventLogger eventLogger
     );
+
+    boolean isInMigrationAge(Federation federation);
+
+    boolean isPastMigrationAge(Federation federation);
```

Comment:
```suggestion
    boolean isInMigrationAge(Federation activeFederation);

    boolean isPastMigrationAge(Federation activeFederation);
```
no big deal but sounds more accurate to me

Reply:
yess, way better. I'll upload the changes in a bit